omxplayer/player.py

```diff
@@ -137,6 +147,12 @@ def wrapped(fun, self, *args, **kwargs):
 
         return decorator(wrapped, fn)
 
+    def load(self, file_path):
+        self._filename = file_path
+        self._load_file(file_path)
+        time.sleep(0.5)  # Wait for the DBus interface to be initialised
```

Comment:
I think we need to extract and promote this to a file constant or a kwarg. On slower Pis this can be insufficient.

Extracting it with a meaningful name will also make the comment superfluous.